articles/101_build_tool.md

```diff
@@ -136,15 +127,15 @@ It automates the generation of CMake config files as well as pkg-config files.
 It additionally provides functions to register different kinds of tests.
 
 A package using `catkin` specifies its meta data in a manifest file named `package.xml`.
-The format of the manifest file is specified in the [ROS REP 140](http://www.ros.org/reps/rep-0140.html).
+The latest format of the manifest file is specified in the [ROS REP 149](http://www.ros.org/reps/rep-0149.html).
 
 ### ament_cmake
 
 [ament_cmake](https://github.com/ament/ament_cmake) is an evolution of `catkin` and is also based on CMake.
 The main difference between `ament_cmake` and `catkin` is described in [another article](http://design.ros2.org/articles/ament.html).
 In the context of the build tool the biggest difference is that `ament_cmake` generates package-specific files to setup the environment to use the package after it has been built and installed.
 
-A package using `ament_cmake` uses the same manifest file as `catkin` (except that it only allows the newer format version 2).
+A package using `ament_cmake` uses the same manifest file as `catkin` (except that it format version 2 or higher).
```

Comment:
missing words in refactoring ?

Reply:
Fixed in 553d408a0762581565bc0aacd4e2032bbde864cd.